chain/network/src/network_protocol/mod.rs

```diff
@@ -591,10 +591,6 @@ impl RoutedMessageBody {
             RoutedMessageBody::ChunkEndorsement(_)
             | RoutedMessageBody::PartialEncodedStateWitness(_)
             | RoutedMessageBody::PartialEncodedStateWitnessForward(_)
-            // TODO(#11099): Remove this when we filter the targets of message at the sender side.
```

Comment:
This should not happen now since we filter out chunk producers from message receivers.